### PR TITLE
Feature/gcn email

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

**Comment by jchate6** [comment]
closes #28

File: helm-chart/values-dev.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ brokers:
 
 hermesFrontEndBaseUrl: https://hermes-dev.lco.global/
 
+hermesEmail: hermes@lco.global
```
Comment (phycodurus) [inline helm-chart/values-dev.yaml:21]: I know the email have to be secret, but for consistency's sake I have `HERMES_EMAIL_USERNAME` and `HEREMS_EMAIL_PASSWORD` defined as k8s secrets (in both dev and prod namespaces). I think keeping these values together and handling them consistently reduces cognitive complexity.

File: helm-chart/templates/_helpers.tpl [hunk 2]
```diff
@@ -119,6 +119,10 @@ build it here and use it everywhere.
   value: {{ .Values.brokers.gcnClassicOverKafka.groupId | quote }}
 - name: HOPSKOTCH_GROUP_ID
   value: {{ .Values.brokers.hopskotch.groupId | quote }}
+- name: HERMES_EMAIL
```
Comment (phycodurus) [inline helm-chart/templates/_helpers.tpl:122]: Please handle in a way consistent with `HERMES_EMAIL_PASSWORD` (see other comments)

File: hermes/views.py [hunk 3]
```diff
@@ -173,11 +186,28 @@ def submit_to_hop(request, message):
...
+    return message_uuid
+
+
+def submit_to_gcn(request, message, message_uuid):
+    # TODO: Add code to submit the message with its message_uuid to gcn here.
+    # First add the uuid into the message, then transfer the message into plaintext format
+    message_plaintext = convert_to_plaintext(message)
+    message_plaintext += '\n\n This message can be viewed at https://hermes.lco.global/messages/' + str(message_uuid)
+    # Then submit the plaintext message to gcn via email
+    send_email(settings.GCN_EMAIL, settings.HERMES_EMAIL, settings.HERMES_EMAIL_PASSWORD,
```
Comment (phycodurus) [inline hermes/views.py:207]: I know the email have to be secret, but for consistency's sake I have `HERMES_EMAIL_USERNAME` and `HEREMS_EMAIL_PASSWORD` defined as k8s secrets. I think keeping these values together and handling them consistently reduces cognitive complexity.

File: hermes_base/settings.py [hunk 4]
```diff
@@ -211,6 +211,10 @@ def str2bool(value):
 SCIMMA_AUTH_BASE_URL = os.getenv('SCIMMA_AUTH_BASE_URL', default='https://my.hop.scimma.org/hopauth')  # for production
 KAFKA_USER_AUTH_GROUP = os.getenv("KAFKA_USER_AUTH_GROUP", default="kafkaUsers")
 
+GCN_EMAIL = os.getenv('GCN_EMAIL', 'circulars@dev.gcn.nasa.gov')
+HERMES_EMAIL = os.getenv('HERMES_EMAIL', 'hermes@lco.global')
```
Comment (phycodurus) [inline hermes_base/settings.py:215]: Please treat these consistently (see other comments).

**Review by phycodurus — CHANGES_REQUESTED** [review]
@cmccully I'm happy to make these changes if you want me to. Just let me know.

**Review by phycodurus — APPROVED** [review]

**Pull request merged**